docs/8-veba-troubleshooting.md

```diff
@@ -0,0 +1,277 @@
+# vCenter Event Broker Appliance - Troubleshooting
+
+## Table of Contents
```

Comment:
Did you use a tool to generate the ToC? Looks like you created it. There are good tools to generate a ToC in vscode which makes it consistent (for other contributors) and easier to generate.

Reply:
I just found a TOC somewhere on Github and copied the way they did it :)

I will look at vscode tools.